changelog/april2024/2024-04-26-apple-silicon-changed-macos14-sonoma-is-now-available.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: macOS14 Sonoma is now available!
+status: changed
+author:
+    fullname: 'Join the #[xxx] channel on Slahttps://scaleway-community.slack.com/channels/apple-siliconck.'
+    url: 'https://slack.scaleway.com'
+date: 2024-04-26
+category: bare-metal
+product: apple-silicon
+---
+
+The version macOS 14 Sonoma is now installed by default on Mac mini M2 (M2-M) and Mac mini M2 pro (M2-L). 
+
+The Mac mini M1 (M1-M) range will stay on macOS 13 by default for now. If you want to enjoy a smooth macOS 14 migration you can opt-in to our latest macOS configuration improvements by reinstalling your server through the console, then performing a manual upgrade using softwareupdate.
```

Comment:
```suggestion
The Mac mini M1 (M1-M) range will stay on macOS 13 by default for now. If you want to enjoy a smooth macOS 14 migration you can opt-in to our latest macOS configuration improvements by reinstalling your server through the console, then performing a manual upgrade using software update.
```

Reply:
`softwareupdate` is the name of the tool. I'll put into formatted code to make it clearer @nerda-codes https://ss64.com/mac/softwareupdate.html

```suggestion
The Mac mini M1 (M1-M) range will stay on macOS 13 by default for now. If you want to enjoy a smooth macOS 14 migration you can opt-in to our latest macOS configuration improvements by reinstalling your server through the console, then performing a manual upgrade using `softwareupdate`.
```